### ISSUE TITLE
get table names: syntax error when blacklist or whitelist is used

### ISSUE THREAD

**Issue by akoeb** [aarondl/sqlboiler-sqlite3#1]
I get following error when I use sqlboiler v3 with this driver:

```
unable to get table names: near "and": syntax error
Error: unable to initialize tables: unable to fetch table data: driver (/home/user/gopath/bin/sqlboiler-sqlite3) exited non-zero: exit status 1
```

I was using following command line:
`$GOPATH/bin/sqlboiler sqlite3`

with following sqlboiler.toml:
```
[sqlite3]
dbname = "database.db"
blacklist = ["goose_db_version"]
```

**Comment by akoeb** [comment]
The error is fixed if you remove the semicolon in line 121 of file `driver/sqlite.go`

**Comment by aarondl** [comment]
Thanks for the PR :) Closed via #2
